hw5/main.py

```diff
@@ -0,0 +1,37 @@
+import math
+import sys
+import statistics
+from my_math import unique_count, unique_mean, index_by_value
+from files import read_csv, write_to_csv
+
+
+def _main():
+    columns, data = read_csv(sys.argv[1])
+    users = [d[3] for d in data]
+    prices = [float(d[2]) for d in data]
+    stores = [d[1] for d in data]
+    items = [d[0] for d in data]
+    print(f'Unique value of stores: {len(unique_count(stores))}')
+    print(f'Unique value of items: {len(unique_count(items))}')
+    print(f'User with max price count: {max(unique_count(users), key=unique_count(users).get)}')
+
+    print(f'~~~Amount of items in each store~~~')
+    stores_count = unique_count(stores)
+    for store in stores_count.keys():
+        print(f'Store {store}: {stores_count[store]}')
```

Comment:
note: the result would be accurate only if there's a single price for each item in store.